### ISSUE TITLE
I'm having issues after trying the same example in the README

### ISSUE THREAD

**Issue by miguel550** [ciceromsjr/myfinancesplugin#2]
When I tried to run the example shown in the README into my app it showed the following error

**Transaction "Sale Bonus" Result => ERROR: Attempt to invoke virtual method 'java.lang.String cicero.minhasfinancas.entity.Conta.e()' on a null object reference.**

Code used

```
@Override
protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    TextView tv = (TextView) findViewById(R.id.sample_text);
    tv.setText(stringFromJNI());

    IncomeTransaction income = new IncomeTransaction();
        income.setDescription("Sale Bonus EXAMPLE");
        income.setAmount(50d);
        income.setConfirmationDate(new Date());
        income.setCreationDate(new Date());
        income.setDueDate(new Date());
        income.setAccount("My Wallet");
        income.setCategory("Others");
        income.setSubcategory("Others");
        income.setRecurrence(Transaction.TransactionRecurrence.NONE); //default


        TransactionSender ts = new TransactionSender.Builder(this)
                .notification(true) //only for incomes and expenses
                .transaction(income)
                .resultReceiver(MyBroadcastReceiver.class)
                .createDependeciesIfNeeded(true)
                .build();

        ts.send();

    }
```

**Comment by ciceromsjr** [comment]
Hey Miguel! I will check fix this up as soon as possible! Thanks for reporting.

**Comment by ciceromsjr** [comment]
Fixed! It will be available on My Finances version 4.6.4.
Library is correct.
